schematics/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="mat-elevation-z8">
+  <mat-table #table [dataSource]="dataSource" matSort aria-label="Elements">
+
+    <!-- Id Column -->
+    <ng-container matColumnDef="id">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>Id</mat-header-cell>
+      <mat-cell *matCellDef="let row">{{row.id}}</mat-cell>
```

Comment:
Can we make it easier for users to know what can be renamed (e.g. `row`)? Perhaps calling the data `element` instead of `row`?